Misc/NEWS.d/next/Library/2018-07-28-14-05-43.bpo-34260.SR5XMW.rst

```diff
@@ -0,0 +1,2 @@
+Fix docstring of ``shutil.copy2()`` to make it more specific about which
```

Comment:
We don't document changes on the documentation. I would suggest to remove the NEWS entry.